martin-mbtiles/src/tile_copier.rs

```diff
@@ -98,23 +139,35 @@ impl TileCopier {
             .execute(&mut conn)
             .await?;
 
-        let schema = query("SELECT sql FROM sourceDb.sqlite_schema WHERE tbl_name IN ('metadata', 'tiles', 'map', 'images')")
-            .fetch_all(&mut conn)
-            .await?;
-
-        for row in &schema {
-            let row: String = row.get(0);
-            query(row.as_str()).execute(&mut conn).await?;
-        }
+        if force_simple {
+            for statement in &["CREATE TABLE metadata (name text, value text);",
+                "CREATE TABLE tiles (zoom_level integer, tile_column integer, tile_row integer, tile_data blob);",
+                "CREATE UNIQUE INDEX name on metadata (name);",
+                "CREATE UNIQUE INDEX tile_index on tiles (zoom_level, tile_column, tile_row);"] {
+                query(statement).execute(&mut conn).await?;
+            }
+        } else {
```

Comment:
Lets add a comment for others to explain this moment:

```suggestion
        } else {
          // DB objects must be created in a specific order: tables, views, triggers, indexes.
          // If there are no triggers, in theory views and indexes can go in any order after tables.
```

In theory, triggers and indexes could be created in any order (i think), but trigger could rely on a view or a table.  I have not seen triggers ever defined on an mbtiles, so we could in theory just filter the type column to be `type IN ('table', 'view', 'index')`.

One more thing - I am not certain your order-by would work - ordered DESC, it would be `view, table, index` - and view cannot be created before the table i would think.

see https://www.sqlite.org/schematab.html